buildout.cfg

```diff
@@ -13,6 +13,10 @@ recipe = zc.recipe.egg
 eggs =
     Sphinx
     docutils
+    ZODB
+    j1m.sphinxautointerface
```

Comment:
Is this better than [repoze.sphinx.autointerface](https://pypi.python.org/pypi/repoze.sphinx.autointerface/)? I ask only because that's what we/I use and I've helped maintain that one in the past.